### PR TITLE
Update Maridia Bug Room

### PR DESCRIPTION
**Pull request by kjbranch**

### CODE REVIEW

File: region/maridia/inner-yellow.json [hunk 1]
```diff
@@ -814,23 +827,60 @@
...
+              "strats": [
+                {
+                  "name": "Kill the Menus",
+                  "notable": false,
+                  "requires": [],
+                  "obstacles": [
+                    {
+                      "id": "A",
+                      "requires": [
+                        {"and":[
```
Comment (osse101) [inline region/maridia/inner-yellow.json:855]: Missing `or`?

File: region/maridia/inner-yellow.json [hunk 2]
```diff
@@ -814,23 +827,60 @@
...
+                          "canCarefulJump",
```
Comment (osse101) [inline region/maridia/inner-yellow.json:856]: is this a `canDodgeWhileShooting`? Or is this a full avoid strat?

File: region/maridia/inner-yellow.json [hunk 3]
```diff
@@ -858,32 +904,22 @@
...
                   "requires": [
                     "h_canTwoTileJumpMorph",
                     {"or":[
-                      "HiJump",
-                      "h_canFly",
-                      "canWalljump",
-                      "canSpringBallJumpMidAir",
-                      "canTrickyDashJump",
-                      "canTrickyUseFrozenEnemies"
+                      {"enemyDamage": { "enemy": "Menu", "type": "contact", "hits": 1 }},
```
Comment (osse101) [inline region/maridia/inner-yellow.json:921]: I think this should require the obstacle clear.
Meaning, put `enemyDamage` as a way to clear the obstacle.

**Review by osse101 — COMMENTED** [review]
Good changes.  This is a hard room.

File: region/maridia/inner-yellow.json [hunk 4]
```diff
@@ -907,8 +939,79 @@
...
+                  "requires": [
+                    {"or":[
+                      "h_canIBJ",
+                      "canWalljump",
+                      "canSpringBallJumpMidAir"
+                    ]},
+                    {"or":[
+                      {"enemyKill":{
+                        "enemies": [ [ "Menu", "Menu", "Menu" ] ],
+                        "explicitWeapons": [
```
Comment (osse101) [inline region/maridia/inner-yellow.json:978]: Missile kill is my first choice.

File: region/maridia/inner-yellow.json [hunk 5]
```diff
@@ -907,8 +939,79 @@
...
+                }
+              ]
+            },
+            {
+              "id": 2,
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": [ "Morph" ]
```
Comment (osse101) [inline region/maridia/inner-yellow.json:1012]: Does this need something for dealing with Menus?
In testing I was blocked by left menu, but never in runs.
Right menu I always kill with a missile, but it looks like you can wait a second for it to pass.

Comment (kjbranch) [reply region/maridia/inner-yellow.json:1012]: I think it is simple enough to be techless. There are probably strats of this difficulty all over. 
The first menu can be jumped over, or lured and then avoided or killed pretty easily.
The right menu can be jumped over, or killed from the left water patch.

File: region/maridia/inner-yellow.json (hunk 4 above)
Comment (kjbranch) [reply region/maridia/inner-yellow.json:978]: Good catch. I was thinking this would be harder to aim. I guess it is really simple though

**Pull request merged**